packages/origin/contracts/Origin/CertificateLogic.sol

```diff
@@ -158,6 +158,39 @@ contract CertificateLogic is CertificateInterface, CertificateSpecificContract,
         }
     }
 
+    /// @notice claims a set of certificates
+    /// @param _idArray the ids of the certificates to be claimed
+    function claimCertificateBulk(uint[] calldata _idArray) external {
```

Comment:
I could limit this to a certain maximum array size like `uint[10] calldata _idArray` or something, to stop overflows on block gas limit.